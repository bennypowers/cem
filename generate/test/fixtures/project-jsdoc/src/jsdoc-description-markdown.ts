/**
 * description
 *
 * the description contains:
 * - markdown list
 * - inline code
 * - multiple lines
 * - markdown headings
 *
 * ## Heading
 * markdown heading follows `list`.
 */
@customElement('jsdoc-description-markdown')
class JsdocDescriptionMarkdown extends LitElement { }
