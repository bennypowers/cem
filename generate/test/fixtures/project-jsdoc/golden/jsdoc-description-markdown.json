{
  "schemaVersion": "2.1.0",
  "modules": [
    {
      "kind": "javascript-module",
      "path": "src/jsdoc-description-markdown.js",
      "declarations": [
        {
          "name": "JsdocDescriptionMarkdown",
          "description": "description\n\nthe description contains:\n- markdown list\n- inline code\n- multiple lines\n- markdown headings\n\n## Heading\nmarkdown heading follows `list`.",
          "superclass": {
            "name": "LitElement",
            "package": "lit"
          },
          "source": {
            "href": "https://github.com/bennypowers/cem/tree/main/generate/test/fixtures/project-yaml/src/jsdoc-description-markdown.ts#L13"
          },
          "kind": "class",
          "tagName": "jsdoc-description-markdown",
          "customElement": true
        }
      ],
      "exports": [
        {
          "kind": "custom-element-definition",
          "name": "jsdoc-description-markdown",
          "declaration": {
            "name": "JsdocDescriptionMarkdown",
            "module": "src/jsdoc-description-markdown.js"
          }
        }
      ]
    }
  ]
}
